test/consolidator/services/DeleteDirServiceSpec.scala

```diff
@@ -56,7 +55,7 @@ class DeleteDirServiceSpec extends AnyWordSpec with Matchers with BeforeAndAfter
 
       whenReady(future) { result =>
         result.isLeft shouldBe true
-        result.left.get.getMessage shouldBe s"$path is not a directory"
+        result.toOption.getOrElse(s"$path is not a directory") shouldBe s"$path is not a directory"
```

Comment:
This is wrong change, it doesn't test `result` as expected, because if result is `Left` then `.toOption` will yield `None` and so left hand side will evaluate to `.getOrElse(s"$path is not a directory")` and this has nothing to do with value in `result`.